src/test/scala-2.11/markets/participants/TestLiquidityDemander.scala

```diff
@@ -34,16 +34,16 @@ case class TestLiquidityDemander(initialDelay: FiniteDuration,
                                  tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
   extends LiquidityDemander[MarketOrderTradingStrategy] {
 
-  val outstandingOrders = mutable.Set.empty[Order]
-
-  val marketOrderTradingStrategy = new TestMarketOrderTradingStrategy
-
   interval match {
     case Some(duration) =>
       context.system.scheduler.schedule(initialDelay, duration, self, SubmitMarketBidOrder)
     case None =>
       context.system.scheduler.scheduleOnce(initialDelay, self, SubmitMarketBidOrder)
   }
+  
+  val outstandingOrders = mutable.Set.empty[Order]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2092014729/issues/source?bid=2921361&fileBranchId=3140879#l44)